eclipse-scout-core/src/dataobject/serialize/DataObjectDeserializer.ts

```diff
@@ -76,14 +79,20 @@ export class DataObjectDeserializer implements DataObjectDeserializerModel, Obje
 export interface DataObjectDeserializerModel extends ObjectModel<DataObjectDeserializer> {
   /**
    * Controls the kind of object that will be created when deserializing unknown DataObjects.
-   * If true, a pojo will be created for unknown DOs. If false, instances of {@link BaseDoEntity} will be created.
-   * Default is false.
+   * If <code>true</code>, a pojo will be created for unknown DOs. If <code>false</code>, instances of {@link BaseDoEntity} will be created.
+   * Default is <code>false</code>.
```

Comment:
I think we use no longer the HTML syntax but the markdown syntax using backticks now?
If we really keep the default fallback for retainTypeVersion to createPojoIfDoIsUnknown (see comment above) then I would clearly indicate this here: Enabling the property createPojoIfDoIsUnknown may also have an effect on the _typeVersion attribute and therefore on the attributes of the deserialized objects. Maybe this side-effect is not expected and it would be better if the default is just 'false'?